<launch>

    <group ns="cam0" clear_params="true">
        <arg name="camera_config" value="AR0134_1280x964_8bit_C.cfg" /> <!-- Config file in camera_config_files folder of your camera. -->
        <arg name="serial_number" value="" /> <!-- Serial number of the camera in the form of "XXXX-XXXX-XXXX", leave it blank "" if there is only one camera. -->
        <arg name="horizontal_flip" value="false" /> <!-- Flip image horizontally. -->
        <arg name="vertical_flip" value="false" /> <!-- Flip image vertically. -->
        <arg name="frame_id" value="cam0" /> <!--Topic frame_id. -->

        <node name="arducam_trigger_ros_node" pkg="arducam_usb2_ros" type="arducam_trigger_ros_node.py" output="screen">
            <param name="config_file" value="$(find arducam_usb2_ros)/camera_config_files/$(arg camera_config)" />
            <param name="camera_serial" value="$(arg serial_number)" />
            <param name="horizontal_flip" value="$(arg horizontal_flip)" />
            <param name="vertical_flip" value="$(arg vertical_flip)" />
            <param name="frame_id" value="$(arg frame_id)" />
        </node>
    </group>

</launch>
